### DISCUSSION TITLE
2008/10/27/fly-on-the-wall

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#248] [Announcements]
# 2008/10/27/fly-on-the-wall

今天，领导让我去列席一些技术讨论会议。参加会议的都是公司比较高层的技术人员，讨论的项目我还没接触过。所以领导也没指望我在会议上发言，他让我趴到会议室墙上去当苍蝇。

https://blog.qizhen.xyz/2008/10/27/fly-on-the-wall

**Comment by ruanqizhen** [comment]
之前博客下的留言：

杰
2008年10月28日 21:57 编辑
呵呵，学了一招，这事我常干，fly on the wall~

回复
hongfei
2008年10月28日 21:15 编辑
默不作声的往往是高手

回复
Mary
2008年10月28日 09:46 编辑
其实观察别人发言更有意思，你只需要对他们微笑着听着，脑子里可以想象，如果这个问题让你来回答的时候怎么能更好，或者他哪里值得你学习的。

回复
逸美
2008年10月28日 09:28 编辑
           呵呵!蛮幽默的.

回复
